test/parsing/errors/labelIneligibleStmt-detailed.good

```diff
@@ -0,0 +1,4 @@
+─── error in labelIneligibleStmt.chpl:1 [LabelIneligibleStmt] ───
+  Cannot label 'FnCall' statement.
+  Only for-, while-do- and do-while-statements can have labels.
```

Comment:
I'd write this as "Only 'for', 'while', and 'do-while' statements can have labels."
Even though `do-while` is strictly speaking not a Chapel syntax element.